src/components/duration.rs

```diff
@@ -857,23 +857,36 @@ impl Duration {
 // ==== Public Duration methods ====
 
 impl Duration {
-    /// Returns the absolute value of `Duration`.
+    /// Determines the sign for the current self.
     #[inline]
     #[must_use]
-    pub fn abs(&self) -> Self {
+    pub fn sign(&self) -> i32 {
+        duration_sign(&self.iter().collect())
+    }
+
+    /// Returns whether the current `Duration` is blank.
+    pub fn blank(&self) -> bool {
+        duration_sign(&self.iter().collect()) == 0
+    }
```

Comment:
Would probably be better to name this `is_zero`, I think.

Reply:
I was more talking about the function itself, since `duration_sign` returns zero only when `self` is zero.

Reply:
Oh, I'm blind lol yeah, that can work too.

Reply:
Glancing back through old temporal meeting minutes. It used to be called `isZero` but was opted off of, but I see no reason to not go with `isZero` as long as the equivalency is noted.

Reply:
Yeah, `blank` is not very descriptive IMHO. We can add some comments when we rename spec methods just like in Boa.